bpf/Makefile

```diff
@@ -170,7 +170,7 @@ testdata:
 	-c ../pkg/alignchecker/testdata/bpf_foo.c \
 	-o ../pkg/alignchecker/testdata/bpf_foo.o
 
-generate: $(BPF)
+generate: $(BPF) bpf_maps.o
```

Comment:
Can we just use bpf2go instead of embedding the .o file in `pkg/maps/registry` to keep things consistent? You should just need to add a new line to `pkg/datapath/bpf/gen.go`. I see below that you have some custom `llvm-strip` stuff going on, but I feel like that could be integrated into bpf2go upstream as another environment option if it's not already supported: `BPF2GO_LLVM_STRIP_FLAGS`.